src/pages/RealtimeMapPage.tsx

```diff
@@ -138,21 +132,21 @@ export default function RealtimeMapPage() {
         </Grid>
         <Grid xs={5}>
           <DateSelector
-            timeValid={moment(from.slice(0, 16))} // remove timezone and seconds
+            timeValid={to} // 
             setTimeValid={(ts) => {
-              const value = ts ? ts.format() : ''
-              setFrom(value)
-              setTo(formatTime(+new Date(value) + (+new Date(to) - +new Date(from)))) // keep the same time difference
+              // const value = ts ? ts.format() : ''
```

Comment:
please remove code in comments

Reply:
We need another PR/issue for the DateSelector & TimeSelector: Now when you set the time to another date or hour, it does not set the state variables to & from accordingly. We need to set "to" to the timeSelected by the user, and from to the number of minutes difference (5 min default, or what the user selected with MinuteSelector component).

Reply:
sounds like it should be solved in this PR, to avoid regressions
also, adding tests to this behaviour would be very beneficial

Reply:
As I was testing it today I realized that the date, hours, and minutes difference selectors were not in sync. I thought we needed another PR to fix it, but I think it's okay now. I made a few changes to make it work. 
Now, when the user picks another date, then the minutes gap he selects will be for the same date.